Cannot build for ios simulator with Apple M1 macbook

### Environment

Using sentry-cordova 1.0.0 for an ionic project with Xcode 13 on a macbook pro with an M1 Processor.

Ionic:

   Ionic CLI                     : 6.18.1 (/usr/local/lib/node_modules/@ionic/cli)
   Ionic Framework               : @ionic/angular 5.9.3
   @angular-devkit/build-angular : 0.803.29
   @angular-devkit/schematics    : 8.3.29
   @angular/cli                  : 8.3.29
   @ionic/angular-toolkit        : 5.0.3

Cordova:

   Cordova CLI       : 11.0.0
   Cordova Platforms : android 8.1.0, ios 6.2.0
   Cordova Plugins   : cordova-plugin-ionic-keyboard 2.1.3, cordova-plugin-ionic-webview 5.0.0, (and 16 other plugins)

Utility:

   cordova-res                          : 0.15.4
   native-run (update available: 1.5.0) : 0.2.9

System:

   ios-sim : 8.0.2
   NodeJS  : v16.13.2 (/usr/local/bin/node)
   npm     : 8.1.2
   OS      : macOS Monterey
   Xcode   : Xcode 13.2.1 Build version 13C100


### Steps to Reproduce

1. Run ionic cordova build ios

### Expected Result

It would build

### Actual Result

ld: warning: building for iOS Simulator, but linking in dylib built for iOS, 'Sentry.framework'
ld: warning: Could not find or use auto-linked framework 'Sentry'
Undefined symbols for architecture arm64:
  "_OBJC_CLASS_$_SentryUser", referenced from:
      objc-class-ref in SentryCordova.o
  "_OBJC_CLASS_$_SentryBreadcrumb", referenced from:
      objc-class-ref in SentryCordova.o
  "_OBJC_CLASS_$_SentryEnvelope", referenced from:
      objc-class-ref in SentryCordova.o
  "_OBJC_CLASS_$_SentryEnvelopeItem", referenced from:
      objc-class-ref in SentryCordova.o
  "_OBJC_CLASS_$_SentryEnvelopeItemHeader", referenced from:
      objc-class-ref in SentryCordova.o
  "_OBJC_CLASS_$_SentryEnvelopeHeader", referenced from:
      objc-class-ref in SentryCordova.o
  "_OBJC_CLASS_$_SentrySdkInfo", referenced from:
      objc-class-ref in SentryCordova.o
  "_OBJC_CLASS_$_SentryId", referenced from:
      objc-class-ref in SentryCordova.o
  "_OBJC_CLASS_$_SentrySDK", referenced from:
      objc-class-ref in SentryCordova.o
  "_OBJC_CLASS_$_SentryOptions", referenced from:
      objc-class-ref in SentryCordova.o
ld: symbol(s) not found for architecture arm64
clang: error: linker command failed with exit code 1 (use -v to see invocation)

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -4,6 +4,7 @@
 
 ### Fixes
 
+- Fix build on Mac M1 ([#262](https://github.com/getsentry/sentry-cordova/pull/262))
 - Support for running with cordova-android 10 ([#246](https://github.com/getsentry/sentry-cordova/pull/246))
 
 ## v1.0.0

--- a/Makefile
+++ b/Makefile
@@ -1,2 +1,2 @@
 build:
-	cd src/ios; carthage update
+	cd src/ios; carthage update --use-xcframeworks; carthage build --use-xcframeworks --no-use-binaries

--- a/plugin.xml
+++ b/plugin.xml
@@ -58,7 +58,7 @@
 
         <!-- We don't have a resource-file here since this adds it to Xcode project -->
 
-        <framework src="src/ios/Carthage/Build/iOS/Sentry.framework" custom="true" embed="true" />
+        <framework src="src/ios/Carthage/Build/Sentry.xcframework" custom="true" embed="true" />
     </platform>
 
     <!-- browser -->

